server/src/test/java/org/opensearch/common/util/ByteArrayBackedBitsetTests.java

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
coming from here: https://github.com/opensearch-project/OpenSearch/pull/14817#discussion_r1704946374

Reply:
Yes thanks for linking